README.md

```diff
@@ -10,10 +10,11 @@ FROM articulate/articulate-node:<tag>
 
 > 🌟 recommended image
 
+* __20-bullseye-slim__ 🌟
 * __18-bullseye-slim__ 🌟
 * __18-lambda__ 🌟
-* __16-bullseye-slim__ 🌟
-* __16-lambda__ 🌟
+* 16-bullseye-slim
+* 16-lambda
```

Comment:
node 16 end of life [is 9/11/2023](https://nodejs.org/en/blog/announcements/nodejs16-eol)